ce_build.sh

```diff
@@ -24,4 +24,7 @@ docker cp $TRAVIS_BUILD_DIR "$ROS_DISTRO"-container:/"$ROS_DISTRO"_ws/src/
 docker exec "$ROS_DISTRO"-container /bin/bash \
   -c "sh /shared/$(basename ${SCRIPT_DIR})/"'ros"$ROS_VERSION"_build.sh' || travis_terminate 1;
 # upload coverage report to codecov
-bash <(curl -s https://codecov.io/bash) -Z -F "${ROS_DISTRO},ROS_${ROS_VERSION}"
+if [ -z "${NO_TEST}" ];
```

Comment:
I understand what you are trying to do here. In the `ros1_build.sh`, we check if `$NO_TEST` is defined => see [here](https://github.com/aws-robotics/travis-scripts/blob/master/ros1_build.sh), if no tests, we won't even try to build test and run `colcon test`.

I just checked the `ce_build.sh` script and it doesn't check for that. I'm okay with that, but just wanted to point it out.

As for this PR, totally agree we can use if/else here, or we can take out the `-Z` option and it will not fail when no test result s are found. 

Please let me know what you think. Thanks.

Reply:
In ce_build.sh we call `ros"$ROS_VERSION"_build.sh` which is either ros1_build.sh or ros2_build.sh, and both have the if for `$NO_TEST`

Regarding the `-Z` option I think it makes sense to use it when tests are enabled, as in the future we'd also want to fail the build if the coverage is below the baseline, using [coverage targets in codecov](https://docs.codecov.io/docs/flags#section-create-custom-notifications)

Reply:
k. LGTM!